sponsorsModule/static/css/sponsor_display.css

```diff
@@ -1,14 +1,50 @@
-.sponsorModule_img{
-    width:100%;
+#sponsors_module{
```

Comment:
Préférablement pas de styling sur un ID

Reply:
Ok merci ! Je ne l'avais pas appris comme ça :/

Reply:
En fait c'est correct en général de target un id en css, mais dans notre cas, théoriquement on pourrais avoir 2 fois le plugin dans la page alors que un id est supposé être unique.

Reply:
> En fait c'est correct en général de target un id en css, mais dans notre cas, théoriquement on pourrais avoir 2 fois le plugin dans la page alors que un id est supposé être unique.

True, mais justement as a general rule of thumb, on style pas sur un ID :P